pylabrobot/resources/hamilton/troughs.py

```diff
@@ -1,55 +1,200 @@
 """Definitions for Hamilton-manufactured Troughs"""
 
+import warnings
+
 from pylabrobot.resources.trough import Trough, TroughBottomType
 
-# # # # # # # # # # Hamilton_1_trough_200ml_Vb # # # # # # # # # #
+# --------------------------------------------------------------------------- #
+# Central empirical calibration data — paired (height_mm, volume_ul)
+# --------------------------------------------------------------------------- #
+
+_TROUGH_CALIBRATIONS: dict[str, dict[str, list[tuple[float, float]]]] = {
+  "hamilton_1_trough_60ml_Vb": {
+    "measurements": [
+      (0.0, 0.0),
+      (2.2, 500.0),
+      (3.5, 1_000.0),
+      (4.0, 1_500.0),
+      (4.7, 2_000.0),
+      (5.2, 2_500.0),
+      (5.6, 3_000.0),
+      (6.0, 3_500.0),
+      (6.3, 4_000.0),
```

Comment:
let's store these closer to the resource definitions rather than in a dictionary far away